packages/terra-site/package.json

```diff
@@ -26,10 +26,11 @@
   "dependencies": {
     "babel-polyfill": "^6.23.0",
     "moment": "^2.17.1",
-    "react": "15.4.2",
-    "react-dom": "15.4.2",
-    "react-intl": "2.2.3",
-    "react-router": "^3.0.2",
+    "prop-types": "^15.5.8",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
```

Comment:
https://semver.npmjs.com/
^ This site is really cool if you want to learn more about sematic versioning with npm.